src/create-element.js

```diff
@@ -68,7 +68,11 @@ export function createVNode(type, props, key, ref) {
 		_parent: null,
 		_depth: 0,
 		_dom: null,
-		_lastDomChild: null,
+		// _lastDomChildSibling must be initialized to undefined b/c it will eventually
+		// be set to dom.nextSibling which can return `null` and it is important
+		// to be able to distinguish between an uninitialized _lastDomChildSibling and
+		// a _lastDomChildSibling that has been set to `null`
+		_lastDomChildSibling: undefined,
```

Comment:
Thinking out loud... would `_nextDom` be a better name for this?

Reply:
tbh that seems preferable.

Reply:
I'll PR this

Reply:
Renamed in #2297